### PR TITLE
Initial pass at features returing in annotations objects for all data…

### LINKED ISSUE #117: Put features into dataset annotations for demo
Features need to come through for each dataset on the `GET /datasets` endpoint.

### PR DESCRIPTION
**Pull request by Sorrento110**
Return features in annotations object for Thursday demo. Closes #117

### CODE REVIEW

File: tds/routers/datasets.py [hunk 1]
```diff
@@ -209,21 +209,55 @@ def delete_qualifier(id: int, rdb: Engine = Depends(request_rdb)) -> str:
...
             session.query(orm.Dataset)
-            .filter(orm.Dataset.simulation_run == is_simulation)
             .order_by(orm.Dataset.id.asc())
             .limit(page_size)
             .offset(page)
             .all()
         )
+        dataset_ids = []
+        for dataset in datasets:
+            dataset_ids.append(dataset.id)
```
Comment (mattprintz) [inline tds/routers/datasets.py:230]:
```suggestion
        dataset_ids = [dataset.id  for dataset in datasets]
```

File: tds/routers/datasets.py [hunk 2]
```diff
@@ -209,21 +209,55 @@ def delete_qualifier(id: int, rdb: Engine = Depends(request_rdb)) -> str:
...
+
+        features = (
+            session.query(orm.Feature)
+            .filter(orm.Feature.dataset_id.in_(dataset_ids))
+            .all()
+        )
+        feature_ids = []
+        for feature in features:
+            feature_ids.append(feature.id)
```
Comment (mattprintz) [inline tds/routers/datasets.py:239]:
```suggestion
        feature_ids = [feature.id for feature in features]
```

**Review by mattprintz — APPROVED** [review]
:+1:

File: tds/routers/datasets.py [hunk 3]
```diff
@@ -209,21 +209,55 @@ def delete_qualifier(id: int, rdb: Engine = Depends(request_rdb)) -> str:
...
 ):
     """
     Get a specific number of datasets
     """
     with Session(rdb) as session:
-        return (
+        if is_simulation:  # Deprecated, makes pylint pass.
+            pass
+        datasets = (
             session.query(orm.Dataset)
```
[the remark this answers is not in the document]
Comment (mattprintz) [reply tds/routers/datasets.py:222]: I don't think the flag is needed on this endpoint.

[the remark this answers is not in the document]
Comment (Sorrento110) [reply tds/routers/datasets.py:222]: So we remove it and if they call with the extra &is_simulation=whatever fast api will just ignore that portion right?

[the remark this answers is not in the document]
Comment (mattprintz) [reply tds/routers/datasets.py:222]: The code as in the branch is just fine. We ignore the param if they include it, but they probably won't and we'll remove the param next week.

[the remark this answers is not in the document]
Comment (Sorrento110) [reply tds/routers/datasets.py:222]: Okay sounds good.

File: tds/routers/datasets.py (hunk 2 above)
Comment (Sorrento110) [reply tds/routers/datasets.py]: changed and pushed.

**Pull request merged**